File: Python3/프로그래머스/0/181919. 콜라츠 수열 만들기/콜라츠 수열 만들기.py
def solution(n):
    answer = []
    
    num = n
    
    while num != 1:
        answer.append(num)
        
        if num % 2 == 0:
            num = num // 2
        else:
            num = 3 * num + 1
    
    answer.append(1)
        
    return answer

File: Swift/프로그래머스/0/181919. 콜라츠 수열 만들기/콜라츠 수열 만들기.swift
import Foundation

func solution(_ n:Int) -> [Int] {
    var answer: [Int] = []
    
    var num = n
    
    while num != 1 {
        answer.append(num)

        
        if num % 2 == 0 {
            num = num / 2
        } else {
            num = 3 * num + 1
        }
    }
    
    answer.append(num)
    
    return answer
}